pkg/projects/Microsoft.NETCore.DotNetHost/Microsoft.NETCore.DotNetAppHost.builds

```diff
@@ -75,6 +79,10 @@
       <OSGroup>Linux</OSGroup>
       <Platform>amd64</Platform>
     </Project>
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroRid)' == 'tizen.4.0.0-armel'" Include="tizen.4.0.0/Microsoft.NETCore.DotNetAppHost.pkgproj">
```

Comment:
I asked a similar question for a similar PR in CoreCLR repo - do we really need Tizen specific entry for this? Can't Tizen use debian armel?